auto2.c

```diff
@@ -601,7 +602,10 @@ int auto2_find_repo()
   }
 
   /* get some files for lazy yast */
-  if(!err) auto2_read_repo_files(config.url.install);
+  if(!err) {
+    if(config.repomd) auto2_read_repomd_files(config.url.install);
+    auto2_read_repo_files(config.url.install);
```

Comment:
so if config.repodm is set, then both reads are called? is it intention? or only read_rpomd should be called?